### ISSUE TITLE
Why is the Time_(s) -1 for each row?

### ISSUE THREAD

**Issue by thoretz** [duderstadt-lab/mars-core#31]
Hello Martians, 

I was trying out the example from the Mars documentation site (https://duderstadt-lab.github.io/mars-docs/tutorials/workingwithmars/create-a-Molecule-Archive/). After the tracking finished I looked at the result. While inspecting the individual entries I saw that the column for Time_(s) was -1 for each row. I added a screenshot for more details. 

![Screenshot 2022-02-02 at 17 04 32](https://user-images.githubusercontent.com/61778269/152192983-c40c8638-b264-4c7c-9144-5063c7d0e152.png)

I was wondering if something went wrong while tracking or if some information is missing so it is presented like that? 

Thanks a lot!

**Comment by karlduderstadt** [comment]
In the current version the -1.0 is added for all rows if the Time_(s) information can't be found in the metadata. I suspect that it might depend on how you opened the image file. This image should have time information. Perhaps you didn't turn on SCIFIO before opening the image.

I am also now thinking, perhaps -1.0 is not the best solution when time information is not found. Instead I would suggest no Time column should be added. I will investigate

**Comment by karlduderstadt** [comment]
I have investigated a bit more. So there are Integers, Floats, Doubles, etc. that need values for 'undefined'

For doubles if could be Double.NaN, but for integer that isn't possible. So I decided to set everything to -1 when the value is not know. This is to ensure consistency. Also, by default numbers in java have a value of 0, but sometimes 0 is a meaningful value. For example, many indexes in the metadata start at 0. Therefore, I decided to use -1.

I have added an if statement to prevent creation of the Time_(s) column is the times values are -1. I hope this fixes the issue and these non-values remain hidden under most circumstances.
